test/test_s3_utils.py

```diff
@@ -409,7 +409,7 @@ def test_s3utils_get_google_key():
     keys = s3u.get_google_key()
     assert isinstance(keys, dict)
     assert keys['type'] == 'service_account'
-    assert keys["project_id"] == "fourdn-fourfront"
+    assert keys["project_id"] == "fourfront-396315"
```

Comment:
Should maybe add inline comment indicating this is a known constant

Reply:
I factored it out as a constant.